src/shared/containers/GetFlags.tsx

```diff
@@ -17,11 +17,12 @@ import {getFlags} from 'src/shared/thunks/flags'
 import {activeFlags} from 'src/shared/selectors/flags'
 import {updateReportingContext} from 'src/cloud/utils/reporting'
 
-type ReduxProps = ConnectedProps<typeof connector>
-type Props = ReduxProps
-
-const GetFlags: FC<Props> = ({status, flags}) => {
+const GetFlags: FC = () => {
   const dispatch = useDispatch()
+  const flags = useSelector(activeFlags)
+  const status = useSelector(
+    (state: AppState) => state.flags.status || RemoteDataState.NotStarted
+  )
```

Comment:
So in the process of trying to figure out why the heck everything was refreshing, I started to poke around here to see if it was the `mstp` (it wasn't), but decided to keep this here since it cleans up the previous implementation which was a mix of two implementations (connect & redux hooks)